courses/free-courses-hi.md

```diff
@@ -356,6 +356,7 @@
 * [Complete Git and GitHub Tutorial](https://www.youtube.com/watch?v=apGV9Kg7ics&t=7s) - Kunal Kushwaha
 * [Complete Git and Github Tutorials for Beginners](https://www.youtube.com/watch?v=Ez8F0nW6S-w) - Apna College
 * [Complete Git and GitHub Tutorials For Beginners In Hindi](https://www.youtube.com/playlist?list=PLzdlNxYnNoafZq1AKcqiGvj0gkzrjmgq7) - Code House
+* [Complete Git in one video](https://youtu.be/tz82ola3oy0?si=hez8NF3ZqatztrdR) - Hitesh choudhary
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
No shortened links.
Do not keep closing PRs that fail the linter. Instead, edit your branch and push the commits!
Once a PR is reviewed, it is disrespectful to close the PR and make a new one, because it won't contain the review comments.

Reply:
I’ve already provided the creator’s name, and since the video is a complete Git learning guide in one video, there isn’t a playlist for it. Let me know if you need any further details!

Reply:
This video is very helpful for anyone who really wants to learn Git, so please merge my PR.
Thank you!